x-pack/filebeat/input/awss3/clean_store.go

```diff
@@ -0,0 +1,81 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package awss3
+
+import (
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/logp"
+
+	"github.com/elastic/beats/v7/libbeat/statestore"
+	"github.com/elastic/go-concert/timed"
+	"github.com/elastic/go-concert/unison"
+)
+
+func cleanStore(canceler unison.Canceler, logger *logp.Logger, store *statestore.Store, states *States, interval time.Duration, objectExpiration time.Duration) {
```

Comment:
nit: maybe add some comment for this function as well?

Reply:
done